### PR TITLE
Revert until fix is found

### PR DESCRIPTION
**Pull request by DevScyu**

### CODE REVIEW

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt [hunk 1]
```diff
@@ -39,7 +38,7 @@ private fun syncBlockLocker(corr: Block) {
  */
 object SectionSyncListener : Listener {
 
-    @EventHandler(ignoreCancelled = true, priority = EventPriority.HIGH)
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
```
Comment (Derongan) [inline src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:41]: Why is this changed to monitor?

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt [hunk 2]
```diff
@@ -113,15 +112,15 @@ object SectionSyncListener : Listener {
...
             block.location.inSectionOverlap
-            && section.isOnTopOf(section2)
+//            && section.isOnTopOf(section2)
             && block.blockData !is Levelled // Water / Lava
         ) isCancelled = true
 
-        if(!section.isOnTopOf(section2)) {
-            block.sync()
-        }
+//        if(!section.isOnTopOf(section2)) {
```
Comment (Derongan) [inline src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:121]: Couldn't we just not register this handler instead of all of this commenting?

**Review by Derongan — CHANGES_REQUESTED** [review]
Please make the description a little bit more informative.

Im assuming this is fixing a sync issue in block overlap regions on block break.

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt (hunk 1 above)
Comment (DevScyu) [reply src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:41]: Ah this can be changed back

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt (hunk 2 above)
Comment (DevScyu) [reply src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:121]: We still want some of the functionality, and this was a temp "fix" until i fixed the bug

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt (hunk 1 above)
Comment (DevScyu) [reply src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:41]: This event does not modify the event and only needs to run after other plugins might have cancelled it

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt (hunk 2 above)
Comment (Derongan) [reply src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:121]: Right, the listener itself is still valid just not this handler. Sounds good. Just fix the monitor level handler above and its good.

**Review by Derongan — APPROVED** [review]
